### DISCUSSION TITLE
Checking a previous release to compare with current version in the repository?

### DISCUSSION THREAD

**Discussion by JonathanUrbach** [r-hub/rhub#626] [Q&A] [answered]
I want to send thanks to the r-hub development team for all their hard work in developing r-hub 2.0.0, particularly the integration with GitHub Actions. Bravo! I think the new system will be spectacularly useful going forward, and I expect it to be one of my favorite platform testing tools.

I was wondering if it was possible to do a check of a previous R package release to compare the results with the current repository, or alternatively check a previous package tarball? This would help determine if a problem detected on a particular platform only (clang-asan) had actually been fixed, or went away due to changes in the dependencies. I attempted to do such a check using the URL argument, and provided the source package URL of an earlier release, but check_rhub used the current repository (as indicated by the package version). I understand this request may be outside of the scope of the r-hub project.

Thanks so much,
Jonathan Urbach

**Comment by gaborcsardi** [comment, marked as answer]
You can create a branch for the previous version, and use the `branch` argument of `rhub_check()`.

**Reply by JonathanUrbach** [reply]
Thank you. I will try that.

**Reply by JonathanUrbach** [reply]
And... it works. Thank you so much!
